### ISSUE TITLE
A bug in cfdUpdateScale?

### ISSUE THREAD

**Issue by MarcusAW** [mhamadAlloush/uFVM#31]
Hello,

When calculating the scales for normalization, assuming that rho and mu are constant, shouldn't phiMax and phiMin just be the values of rho and mu, or could I have a misunderstanding? 

Thanks,
Marcus

**Comment by mmalloush** [comment]
yes if rho is constant, its scale, min and max numbers will be the same as
rho.
